test/test_feature.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+
+import sbol3
+
+class TestFeature(unittest.TestCase):
+    
+    def setUp(self) -> None:
+        sbol3.set_defaults()
+
+    def tearDown(self) -> None:
+        sbol3.set_defaults()
+
+    def test_create(self):
```

Comment:
This test does not run validation yet. After you add the SubComponent you'll want to run validation. You'll probably also want to ensure that a valid `Feature` returns appropriate from validation, and then test an invalid `Feature` (really `SubComponent`) and ensure that the validation method properly detects that the `Feature` is not valid.